CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing Guidelines
+The Kubernetes Dashboard Project is a web-based UI for Kubernetes clusters.It allows users to manage the cluster as well as application running in the cluster.All the contribution is done via github.Please also read the contributors guide
+https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
+
+### Sign the CLA
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests. Please see https://github.com/kubernetes/community/blob/master/CLA.md for more info
+
+### Learn a bit about the kubernetes dashboard
+Before managing and running application in kubernetes dashboard it is recommended to go through the doc 
```

Comment:
Capitalize Kubernetes and Dashboard.

Maybe rephrase? "To learn more about the Kubernetes Dashboard project, please read through the user documentation:"